packaging/Dockerfile.centos_7_stable_rpm

```diff
@@ -0,0 +1,19 @@
+# This Dockerfile is based on the recommendations provided in the
```

Comment:
How about `latest-epel-testing`

Reply:
`epel-testing` is IMHO implementation detail. The idea is to provide the stable/tagged nmstate releases via docker and epel-testing is current repo that is used for this.

Reply:
`rpm` is also an implementation detail.
`latest-release`, just `latest` or just leave the stable then?

Reply:
Please see my answer in the conversaion.